### PR TITLE
fix: broken links

### PR DESCRIPTION
**Pull request by torrespro**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -87,7 +87,7 @@ For Eclipse the process requires loading the formatting style and the import ord
 </details> 
 
 [openhospital]: https://www.open-hospital.org/
-[core]: https://github.com/informatici/openhospital/openhospital-core
+[core]: https://github.com/informatici/openhospital-core
```
Comment (mwithi) [inline README.md:90]: The problem was only here, in the reusable link (https://github.com/informatici/openhospital-core); with this fixed, you can leave unchannged the other ones.

Thanks!

**Review by mwithi — CHANGES_REQUESTED** [review]
Only one line needs fix

**Comment by torrespro** [comment]
https://github.com/informatici/openhospital-api/pull/117

**Comment by torrespro** [comment]
https://github.com/informatici/openhospital-api/pull/117

**Pull request closed without merging**